packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -76,10 +77,28 @@ const JurisdictionModal = ({
                 : available_account.shortcode !== 'maltainvest')
     );
 
+    const swapfree_available_accounts = trading_platform_available_accounts.filter(
+        available_account =>
+            available_account.market_type === 'all' &&
+            (show_eu_related_content
+                ? available_account.shortcode === 'maltainvest'
+                : available_account.shortcode !== 'maltainvest')
+    );
+
+    const getMT5Title = (type: string) => {
+        if (type === 'synthetic') {
```

Comment:
this is being duplicated everywhere! also use CFD_text

Reply:
Done